doc/reference/actor.md

```diff
@@ -2,755 +2,738 @@
-::: tip usage
+### actor-server-identifier
+```scheme
+(actor-server-identifier (srv (current-actor-server)))
 ```
-(protocol-info-extend ...)
-```
-:::
 
-Please document me!
+Returns an actor server's identifier.
 
-### protocol-info-calls
-::: tip usage
-```
-(protocol-info-calls ...)
+### register-actor!
+```scheme
+(register-actor! name (srv (current-actor-server)))
 ```
-:::
 
-Please document me!
+Registers the current thread in an actor server as an actor with the name `name`.
 
-### protocol-info-events
-::: tip usage
-```
-(protocol-info-events ...)
+### connect-to-server!
+```scheme
+(connect-to-server! id (addrs #f) (srv (current-actor-server)))
 ```
-:::
 
-Please document me!
+Instructs an actor server to connect to another server.
+- `id` is the identifier of the target server
+- `addrs` is an optional list of addresses; if none is specified and
+  the server is uknown, its addresses will be resolved through the
+  ensemble registry.
 
-### protocol-info-streams
-::: tip usage
+### list-actors
+```scheme
+(list-actors (srv (current-actor-server)))
 ```
-(protocol-info-streams ...)
-```
-:::
-
-Please document me!
 
+Lists the actors registered with an actor server.
+Returns a list of references.
 
-## RPC
-
-### current-rpc-server
-::: tip usage
-```
-(current-rpc-server ...)
+### list-connections
+```scheme
+(list-connections (srv (current-actor-server)))
 ```
-:::
 
-Please document me!
+Lists the current connections of an actor server.
+Returns an associative list, with the server identifier at the car and the
+list of addresses connected at the cdr.
 
-### start-rpc-server!
-::: tip usage
+### default-known-servers
+```scheme
+(default-known-servers)
 ```
-(start-rpc-server! ...)
-```
-:::
 
-Please document me!
+Returns a hash table with the default known servers.
+By default it only contains the registry reachable in its default address.
 
-### stop-rpc-server!
-::: tip usage
-```
-(stop-rpc-server! ...)
+### set-default-known-servers!
+```scheme
+(set-default-known-servers! servers)
 ```
-:::
 
-Please document me!
+Sets the default known servers; `servers` must be a hash table mapping
+server identifiers to lists of addresses.
 
-### rpc-connect
-::: tip usage
-```
-(rpc-connect ...)
+### default-registry-addresses
+```scheme
+(default-registry-addresses)
 ```
-:::
 
-Please document me!
+Returns the default registry addresses, as a list of addresses.  By
+default the registry is reachable at `/tmp/ensemble/registry` in the
+current host.
 
-### rpc-register
-::: tip usage
-```
-(rpc-register ...)
+### set-default-registry-addresses!
+```scheme
+(set-default-registry-addresses! addrs)
 ```
-:::
 
-Please document me!
+Sets the default registry addresses.
 
-### rpc-unregister
-::: tip usage
+### server-address-cache-ttl
+```scheme
+(server-address-cache-ttl)
 ```
-(rpc-unregister ...)
-```
-:::
 
-Please document me!
+Returns the actor's server address cache TTL in seconds; by default this is 5 minutes.
```

Comment:
fixnum only? flonum? why 5 min?

Reply:
real.

5min looks fine to me, feel free to suggest a better number. I don't think we need to bother justifying those numbers in the documentation, it gets into _too much detail_ territory.